.travis.yml

```diff
@@ -7,6 +7,7 @@ notifications:
   email: false
 node_js:
   - 'node'
+  - '7'
```

Comment:
we should definitely include 4, since it's still LTS. 7 doesn't really matter, but should work.